crates/worker/src/partition_processor_manager.rs

```diff
@@ -962,10 +974,10 @@ impl PartitionProcessorManager {
                 }
             }
             Entry::Occupied(pending) => {
-                info!(
+                debug!(
                     %partition_id,
                     snapshot_id = %pending.get().snapshot_id,
-                    "A snapshot export is already in progress, refusing to start a new export"
+                    "A snapshot export is currently underway"
```

Comment:
I +1 moving to debug, but I liked the clear messaging of the previous one. It calls out the "action" or lack thereof.

Reply:
Thanks for the feedback!